### DISCUSSION TITLE
TSML UI Width

### DISCUSSION THREAD

**Discussion by jimmedlen** [code4recovery/12-step-meeting-list#1380] [Q&A]
How do I set the width of the TSML UI ?  The UI is 100% width on the page. I would like to have Left and right margins or Padding,
some white space to the left and right of the Meeting Guide 

The Legacy UI was centered with white space on the left and right. When I select TSML UI the Meeting List goes to 100%. of the page and seems to ignore the parent theme and any CSS I add to the page or site.

**Comment by joshreisner** [comment]
could we see a URL on your site with TSML UI on it? 

hard to say in isolation, because every theme is dfferent. but i would try

```css
#tsml-ui {
  margin: 0 auto;
  max-width: 1200px; 
}
```

**Comment by jimmedlen** [comment]
Our main meeting page URLhttps://area22indiana.org/meetings/Thanks,Jim ***@***.*** Mar 7, 2024, at 10:11 AM, Josh Reisner ***@***.***> wrote:﻿
could we see a URL on your site with TSML UI on it?
hard to say in isolation, because every theme is dfferent. but i would try
#tsml-ui {
  margin: 0 auto;
  max-width: 1200px; 
}
